src/components/home/partners/CallToAction.jsx

```diff
@@ -1,8 +1,45 @@
+import Button from '@mui/material/Button';
+import Container from '@mui/material/Container';
+import Grid from '@mui/material/Grid';
+import Typography from '@mui/material/Typography';
+
 const CallToAction = () => {
-  // any logic needed
-  // return (
-  // all the JSX (looks like HTML) to render
-  // )
+  return (
+    <Container maxWidth="xl" sx={{ pt: '50px', pb: '100px' }}>
+      <Grid container spacing={1} alignItems="center">
+        <Grid item xs={12} md={8}>
+          <Typography variant="h4" pb={'2rem'}>
+            Empower Change: Become a partner in our civic tech movement.
+          </Typography>
+          <Typography variant="h4">
+            Have an idea for a civic project in Portland or want to partner with us?
+          </Typography>
+        </Grid>
+        <Grid item xs={12} md={4} style={{ textAlign: 'center', alignItems: 'center' }}>
+          <Button
+            sx={{
```

Comment:
This isn't doing anything, needs to be on the `Grid item`

add `mt={{ xs: '50px', md: '0' }}` to line 18 to see.